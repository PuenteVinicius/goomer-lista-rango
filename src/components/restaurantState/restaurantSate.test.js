import React from 'react';
import RestaurantState from './restaurantState';
import { mount } from 'enzyme';
import toJson from "enzyme-to-json"

it("should render the homepage for the non logged in user", () => {
  const props = {
    hours: [
      {
        from: "09:00",
        to: "18:00",
        days:[2,3,4,5,6]
      },
      {
        from: "11:00",
        to: "20:00",
        days:[7,1]
      }
    ]
  }
  const app = mount(<RestaurantState value={props.hours} />)
  expect(toJson(app)).toMatchSnapshot()
})