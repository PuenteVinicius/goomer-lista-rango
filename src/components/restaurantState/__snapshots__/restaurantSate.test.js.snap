// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the homepage for the non logged in user 1`] = `
<_default
  value={
    Array [
      Object {
        "days": Array [
          2,
          3,
          4,
          5,
          6,
        ],
        "from": "09:00",
        "to": "18:00",
      },
      Object {
        "days": Array [
          7,
          1,
        ],
        "from": "11:00",
        "to": "20:00",
      },
    ]
  }
>
  <div
    className="restaurant-state restaurant-state--open"
  >
    <span
      className="restaurant-state__text"
    >
      Aberto Agora
    </span>
  </div>
</_default>
`;
